src/App.js

```diff
@@ -6,6 +6,8 @@ import { fetchBusinesses } from './services/yelp';
 function App() {
   const [businesses, setBusinesses] = useState([]);
   const [loading, setLoading] = useState(true);
+  const [zip, setZip] = useState('54302');
```

Comment:
Setting default state here doesn't do anything for the initial search, since there s a default Zip Code declared inside the `yelp.js` file